app/views/users/index.html.erb

```diff
@@ -1 +1,21 @@
-<h1>Here is the list of all users</h1>
\ No newline at end of file
+<div class="container">
+    <% @users.each do |user| %>
+        <div class="card mb-3" style="max-width: 800px;">
+            <div class="row g-0">
+                <div class="col-md-4">
+                    <img src =<%= user.photo %> class="img-fluid rounded" alt="Profile picture" style="height: 200px; width: 200px;">
+                </div>
+                <div class="col-md-4">
+                    <div class="card-body">
+                        <h5 class="card-title"><%= link_to user.name, user_path(user), style: "text-decoration: none;" %></h5>
+                    </div>
```

Comment:
- [ ] Kindly style the image to be stand alone as it is in the wireframe. Yours is connected which is not same as the given design  👍 
### Your work:
![image](https://github.com/zuhebahmed88091/Blog_App_Rails/assets/50990925/3bd28616-bd90-4997-a03e-1ef227fe8cb9)

### Wireframe:
![image](https://github.com/zuhebahmed88091/Blog_App_Rails/assets/50990925/6b20c0bb-de14-4bb4-b153-c3c2f354bd39)

- [Optional] Consider using the `...._user.html.erb` partial for your view. this is an industry practice, and makes you a better developer

```
    # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```

Reply:
fixed